lib/formats/putty.js

```diff
@@ -52,10 +57,76 @@ function read(buf, options) {
 	if (key.type !== keyType) {
 		throw (new Error('Outer key algorithm mismatch'));
 	}
+
+	si += publicLines;
+	if (lines[si]) {
+		parts = splitHeader(lines[si++]);
+		assert.equal(parts[0].toLowerCase(), 'private-lines');
+		var privateLines = parseInt(parts[1], 10);
+		if (!isFinite(privateLines) || privateLines < 0 ||
+			privateLines > lines.length) {
```

Comment:
style nit: you've used a tab for continuation here (the rest of the repo is tab indents, but 4-space continuation -- it's a bit of a weird style in javascript, I know, comes from the kernel C style on BSD/illumos)

same applies to other continuations in here (I won't comment on all of them one by one)